nengo_gui/gui.py

```diff
@@ -1,111 +1,103 @@
-import os
-import warnings
+"""Classes to instantiate and manage the life cycle of the nengo_gui
+backend."""
 
-import nengo_gui.page
-import nengo_gui.server
+from __future__ import print_function
+
+import select
+import signal
+import sys
+
+import nengo_gui
+from nengo_gui.guibackend import GuiServer
+
+
+class ServerShutdown(Exception):
+    """Causes the server to shutdown when raised."""
+    pass
 
 
 class GUI(object):
```

Comment:
Noting that this is what you should use if you want to start the server from the commandline would be helpful. Otherwise the fact that it's a class is a bit surprising.